plugins/InspectionGadgets/test/com/siyeh/igtest/controlflow/negated_conditional_expression/NegatedConditionalExpression.java

```diff
@@ -2,13 +2,21 @@
 
 class NegatedConditionalExpression {
 
-    String myField1 = null;
-    String myField2 = null;
-
-    boolean x(NegatedConditionalExpression myClass) {
-      System.out.println(!(myClass.myField2 != null));
-      System.out.println(myField2 != null ? !myField2.equals(myClass.myField2) : myClass.myField2 != null);
-      final boolean b = myField1 != null ;
-      return <warning descr="Negating conditional expression">!</warning>(myField2 != null ? !myField2.equals(myClass.myField2) : myClass.myField2 != null);
-    }
-}
\ No newline at end of file
+  String myField1 = null;
```

Comment:
This is the data file of the test of a different inspection. NegatedConditionalInspection does not yet have a test. Could you create one and undo this change?